### PR TITLE
Convert print statements to logging statements

### PR DESCRIPTION
**Pull request by amarkpayne**
Remove print statements in main.py and library.py that would otherwise print regardless of logging level.

### CODE REVIEW

**Comment by mliu49** [comment]
Haha, I was also in the middle of preparing a pull request for these print statements.

**Comment by amarkpayne** [comment]
The only thing I wasn't sure about was the logging level to assign. Let me know if you think any of the levels should be changed.

File: rmgpy/data/kinetics/library.py [hunk 1]
```diff
@@ -171,23 +171,23 @@ def convertDuplicatesToMulti(self):
...
+        logging.info("trying to find duplicates")
         entries_to_remove = []
         for entry0 in self.entries.values():
             if entry0 in entries_to_remove:
                 continue
             reaction0 = entry0.item
             if not reaction0.duplicate:
                 continue
-            print "Found a duplicate reaction: {0}".format(reaction0)
+            logging.info("Found a duplicate reaction: {0}".format(reaction0))
```
Comment (mjohnson541) [inline rmgpy/data/kinetics/library.py]: I think this one and the one above it can be debug

File: rmgpy/data/kinetics/library.py [hunk 2]
```diff
@@ -224,9 +224,9 @@ def convertDuplicatesToMulti(self):
             entry0.longDesc = longDesc
             entries_to_remove.extend(duplicates[1:])
         for entry in entries_to_remove:
-            print "removing duplicate reaction with index {0}.".format(entry.index)
+            logging.info("removing duplicate reaction with index {0}.".format(entry.index))
```
Comment (mjohnson541) [inline rmgpy/data/kinetics/library.py]: I think this one can be debug too

File: rmgpy/data/kinetics/library.py [hunk 3]
```diff
@@ -171,23 +171,23 @@ def convertDuplicatesToMulti(self):
...
+            logging.info("Found a duplicate reaction: {0}".format(reaction0))
             duplicates = [entry0]
             for entry in self.entries.values():
                 reaction = entry.item
                 if reaction0 is reaction:
                     continue
                 if reaction0.isIsomorphic(reaction, eitherDirection=False):
                     if reaction0.reversible != reaction.reversible:
-                        print "Reactions isomorphic but with different reversibilities"
+                        logging.info("Reactions isomorphic but with different reversibilities")
```
Comment (mjohnson541) [inline rmgpy/data/kinetics/library.py]: @mliu49 , @goldmanm how important is this one?  Should it be debug or info?

Comment (mliu49) [inline rmgpy/data/kinetics/library.py]: I marked all of these as debug in [my branch](https://github.com/mliu49/RMG-Py/commit/51e5e5727e8116ecd4af5cf4e84228a6bd54feb4).

Comment (amarkpayne) [inline rmgpy/data/kinetics/library.py]: I made the changes on my local branch, I'll push it to official after everything gets settled.

**Comment by mliu49** [comment]
I added a couple more commits from what I was working on.

**Pull request merged**